module-2/backend/week_two.md

```diff
@@ -5,16 +5,36 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  - it is a bridge between your ruby code and the sql database
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+    class - they apply to every instance in that class
```

Comment:
This is a stale version of the questions. Be sure to set the Turing repo up as an [upstream repository](https://help.github.com/articles/configuring-a-remote-for-a-fork/), and then [fetch the latest version](https://help.github.com/articles/syncing-a-fork/) before you start working on the questions for the week.